text/setup.py

```diff
@@ -22,9 +22,24 @@
     'scipy',
     'pandas',
     'scikit-learn',
-
+    'fairscale',
+    'scikit-image',
+    'boto3',
+    'pytest',
+    'smart_open',
+    'pytorch_lightning>=1.5.9',
+    'torchmetrics',
+    'transformers>=4.16.2',
+    'Pillow',
+    'tqdm',
+    'pandas',
+    'nptyping',
+    'omegaconf',
+    'timm',
+    'pytest',
+    'sentencepiece',
```

Comment:
[!!!] This is extremely fragile. We need version caps in place.

Why do we depend on so much? We should not depend on timm, Pillow, etc. in text.